package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "recette",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "test": "jest"
+  },
+  "dependencies": {
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
**chore** 🧹
Jest is not required in production environment
```suggestion
```

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>